SingularityUI/app/components/requestDetail/header/RequestAlerts.jsx

```diff
@@ -38,7 +38,10 @@ const RequestAlerts = ({requestId, requestAPI, bounces, activeTasksForRequest, d
   const { pendingDeploy, activeDeploy } = requestParent;
   if (pendingDeploy) {
     const deployingInstanceCount = Utils.request.deployingInstanceCount(requestParent, activeTasksForRequest.data);
-    const { instances } = requestParent.request;
+    let instances = equestParent.request.instances;
```

Comment:
`requestParent`?

Reply:
yep, thanks, hadn't actually gotten a chance to test this one yesterday